### PR TITLE
Improve virtualenv and framework builds

### PR DESCRIPTION
**Pull request by jenshnielsen**
Document to use pythonw on conda and reorganise the docs a bit by splitting it into two sections

### CODE REVIEW

**Comment by jenshnielsen** [comment]
Closes #6746

File: doc/faq/virtualenv_faq.rst [hunk 1]
```diff
@@ -40,6 +43,7 @@ exist. Some of these are given here:
 * The ``TKAgg`` backend doesn't require any external dependencies and is
   normally always available.
 * The ``QT4`` framework ``PySide`` is pip installable.
+* ``PYQT5`` is pip installable on Python 3.5.
 * The upcoming `WX Phoenix <http://wiki.wxpython.org/ProjectPhoenix>`_ toolkit
   is ``pip`` installable.
```
Comment (efiring) [inline doc/faq/virtualenv_faq.rst]: I know it is not part of your changeset, but this line about WX is highly misleading, as the Pypi link shows:
https://pypi.python.org/pypi/wx/3.0.3
It is unofficial, from a snapshot, and it requires lots of arcane dependencies.  I don't think we should call it "pip installable" without at least some qualification.

Comment (jenshnielsen) [inline doc/faq/virtualenv_faq.rst]: Agreed, This was originally written under the assumption that a release of WX Phoenix was to happen soon. Evidently that is not the case. We can add it again once the release happens

File: doc/faq/osx_framework.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+build. However, the Conda developers have made it easy to install
+a framework build in both the main environment and in Conda envs.
+To use this install python.app ``conda install python.app`` and
+use ``pythonw`` rather than ``python``
+
+
+Long version
+============
+
+Unfortunately virtualenv creates a non
```
Comment (matthew-brett) [inline doc/faq/osx_framework.rst]: Worth another mention of `venv` here in case someone skipped straight to the long version?

**Pull request merged**